qemu/seabios_iommu.py

```diff
@@ -0,0 +1,111 @@
+import logging
+
+from virttest import error_context
+from virttest import utils_misc
+from virttest import utils_test
+from virttest import utils_disk
+from provider.storage_benchmark import generate_instance
+
+
+def check_data_disks(test, params, env, vm, session):
+    """
+    Check guest data disks (except image1)
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    :param vm: VM object
+    :param session: VM session
+    """
+    image_list = params.objects("images")
```

Comment:
better to directly get the sublist via `image_list = params.objects("images")[1:]`

Reply:
Updated, Thanks!